google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -515,6 +563,30 @@ private Builder() {
                               .build())
                       .build());
 
+      long numOfChannels = 2 * Runtime.getRuntime().availableProcessors();
+      bulkReadRowsSettings =
+          BigtableBulkReadRowsCallSettings.newBuilder(new ReadRowsBatchingDescriptor())
+              .setRetryableCodes(readRowSettings.getRetryableCodes())
```

Comment:
lease add a precondition.checkstate to EnhancedBigtableStubSettings(Builder builder) that makes sure that the codes here match `.readRowSettings.getRetryableCodes()`